test/sequential/test-deprecation-flags.js

```diff
@@ -16,8 +16,7 @@ execFile(node, normal, function(er, stdout, stderr) {
   console.error('normal: show deprecation warning');
   assert.equal(er, null);
   assert.equal(stdout, '');
-  assert.equal(stderr, '(node) util.debug is deprecated. Use console.error ' +
-                       'instead.\nDEBUG: This is deprecated\n');
+  assert(/util\.debug is deprecated.../.test(stderr));
```

Comment:
Heh. I put `...` in my original comment to avoid typing out the entire message. Sorry for the mix up.

Funny, this should actually still work, since . matches a single character. I don't think it's necessary to try to match the entire deprecation message verbatim. Drop the `...` and this LGTM. Sorry again.